agent/definitions.py

```diff
@@ -8,6 +8,8 @@
 
 from agent.exploits import common
 
+from packaging import version
```

Comment:
is this a Python package? if so it should be above the `Ostorlab` modules

Reply:
fixed